design-system-docs/src/_guides/overview.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Overview
+order: 1
+---
+
+The Citizens Advice Design System is a set of accessible and reusable foundations, and components used across our digital products.
+
+The design system guidance is broadly split into two main areas:
+
+- [Foundations](/foundations) - Core design foundations including colours, fonts, spacing, grids, and an icon system.
+- [Components](/components) — Individual components that make up the bulk of the design system. Includes lower-level components such as buttons and tables up to larger patterns like our global header or footer.
+
+## Using the Design System
+
+The design system is implemented and distributed as a component library made up of two parts:
+
+- An npm package containing styles, JavaScript modules, fonts, and icons.
+- A companion ruby gem which provides a way of using design system components in a Rails app.
+
+### NPM package
+
+The central part of the design system is the `@citizensadvice/design-system` package.
+
+At its heart the design system is a set of CSS styles, HTML patterns, and minimal JavaScript behaviour.
+
+The `@citizensadvice/design-system` package bundles the following:
+
+- A set of SCSS files
+- A set of JavaScript modules
+- Our brand font
+- A set of custom icons (as an icon font and SVGs)
+
+You can install the core package from npm. We recommend pinning to an exact version. If you are also using the Rails components these versions need to match.
+
+```sh
+npm install --save-exact @citizensadvice/design-system@5.5.0
+```
+
+This package—alongside companion HTML patterns—is the minimum requirement when using the design system.
+
+#### CSS
+
+The recommended way of using CSS from the design system is through SCSS.
+
+The styles can be loaded by adding the following to your SCSS stylesheet:
+
+```scss
+@import '@citizensadvice/design-system/scss/lib.scss';
+```
+
+The design system requires the use of [Dart Sass](https://sass-lang.com/dart-sass).
+
+The distributed package includes compiled CSS in `lib/lib.css` if you are not using SCSS.
+
+#### Web fonts
+
+The design system uses distributes two sets of fonts. Open Sans, our brand font, and a custom icon font. These are included with the npm package.
```

Comment:
The design system **_uses distributes_** two sets of fonts.

This doesn't read right to me